dev/mima

```diff
@@ -24,6 +24,9 @@ set -e
 FWDIR="$(cd "`dirname "$0"`"/..; pwd)"
 cd "$FWDIR"
 
+# Skip maven local resolver.
+export SKIP_LOCAL_M2=true
```

Comment:
We make `dev/mima` ignore the `local maven cache` directly, so that the environment and process that all tests depend on are consistent in both `GA` and `local`.